packages/bpk-theming/src/BpkThemeProvider.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ThemeProvider } from 'theming';
+
+import createWithTheme from './createWithTheme';
+import withCssVariables from './withCssVariables';
+
+const BpkThemeProvider = withCssVariables(ThemeProvider);
+
+const disableThemingForComponent = () => {
```

Comment:
What's the thinking behind `disableThemingForComponent`?

Reply:
Ah yeah, didn't mean to push this yet 😅 

The idea was that a component that's not given the correct theme attributes could call it and it would remove the CSS variables from `BpkThemeProvider`, turning off theming for that component.

So let's say `BpkText` needs `textColor` and `textBackgroundColor`, and only one is supplied. The component could call that method and `--text-color` and `--text-background-color` would be unset.